src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,31 @@
+package duke.tasks;
+
+import duke.exceptions.UserInputException;
+
+import static duke.exceptions.UserInputException.inputExceptionType.*;
+
+public class Event extends Task {
+    protected String startTime;
+    protected String endTime;
+
+    public Event(String newTaskInfo) throws UserInputException {
+        final String[] eventSplit = newTaskInfo.trim().split("/+", 3);
```

Comment:
Is there other better way of interpreting the user input? What could happen here when user enter a description with spaces? Just a food for thought if you have more time for the project.